src/components/Modal/style.scss

```diff
@@ -0,0 +1,122 @@
+@import "../../styles/vars.scss";
+@import "../../styles/fonts.scss";
+@import "../../styles/breakpoints.scss";
+
+.modal-wrapper{
+    position: fixed;
+    height: 100%;
+    width: 100%;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 1000;
+    overflow: auto;
+    text-align: center;
+
+    .modal{
+        display: inline-block;
+        margin: 120px;
+        padding: 20px;
+        background: #fff;
+        overflow: auto;
+        width: auto;
+        height: auto;
+        border: 0.8px solid $gray3;
+        text-align: center;
+        padding: 0.2vw 1vw;
+        background-color: $gray-background;
+        opacity: 100%;
+        border-radius: 5px;
+        font-size: 10pt;
+        p {
+            padding: 2px;
+        }
+        .button-wrapper{
+            width: 100%;
+            align-content: right;
+            display: inline-block;
+        }
+        
+        .close-button{
+            float: right;
+            width:20px;
+            height: 20px;
+            background-color: white;
+            text-align: center;
+        }
+
+        .close-button:hover{
+            background-color: $gray2;
+        }
+    }   
+}
+
+.inline-modal-wrapper{
+    position: fixed;
+    height: auto;
+    width: auto;
+    z-index: 1000;
+    overflow: auto;
+    text-align: center;
+    transform: translate(-50%, -50%);
```

Comment:
The `transform` property actually needs additional support for other browsers, can you add some vendor prefixes? 
Example:

```
 -webkit-transform: rotate(30deg); /* Ch <36, Saf 5.1+, iOS < 9.2, An =<4.4.4 */
      -ms-transform: rotate(30deg); /* IE 9 */
          transform: rotate(30deg); /* IE 10, Fx 16+, Op 12.1+ */
```

Reply:
Wow I had no idea something like this existed. I just read up on it now! Thanks!